src/com/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -57,7 +57,9 @@
             [com.puppetlabs.puppetdb.config :as conf]
             [com.puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.kitchensink.core :as kitchensink]
+            [robert.hooke :as rh]
             [puppetlabs.trapperkeeper.core :refer [defservice main]]
+            [puppetlabs.trapperkeeper.services :refer [service-id]]
```

Comment:
service-id, interestingly, is not automatically introduced by defservice. Perhaps an older TK did it that way.